Source/Extensions/Blazorise.Video/Video.razor.cs

```diff
@@ -613,11 +629,26 @@ public Task NotifyReady()
     /// </summary>
     [Parameter] public string[] ControlsList { get; set; } = new string[] { VideoControlsType.PlayLarge, VideoControlsType.Play, VideoControlsType.Progress, VideoControlsType.CurrentTime, VideoControlsType.Mute, VideoControlsType.Volume, VideoControlsType.Captions, VideoControlsType.Settings, VideoControlsType.Pip, VideoControlsType.Airplay, VideoControlsType.Fullscreen };
 
+    /// <summary>
+    /// If the default controls are used, you can specify which settings to show in the menu.
+    /// </summary>
+    [Parameter] public string[] SettingsList { get; set; } = new string[] { VideoSettingsType.Captions, VideoSettingsType.Quality, VideoSettingsType.Speed, VideoSettingsType.Loop };
```

Comment:
Well, would be cool to have enum flags for this maybe? Would that not make the user api better?

Reply:
I guess its a good idea.